xc7/boards.cmake

```diff
@@ -1,10 +1,19 @@
 get_target_property_required(OPENOCD env OPENOCD)
 get_target_property_required(OPENOCD_TARGET env OPENOCD_TARGET)
 
+#define_board(
```

Comment:
You don't need to comment out other boards, especially if they use different devices.